define/types.go

```diff
@@ -267,14 +267,14 @@ func cloneToDirectory(url, dir string) ([]byte, string, error) {
 	cmd = exec.Command("git", "init", dir)
 	combinedOutput, err := cmd.CombinedOutput()
 	if err != nil {
-		return combinedOutput, gitSubdir, fmt.Errorf("failed while performing `git init`: %w", err)
+		return combinedOutput, gitSubdir, fmt.Errorf("failed while performing `git init`: %s", err.Error())
```

Comment:
It would likely makes sense to add a comment here (once at least for all four cases) why this is done as this seems like an anti pattern.

Reply:
Done.